### ISSUE TITLE
Prisma init fails (related to brew)

### ISSUE THREAD

**Issue by covertbert** [prisma/prisma#3150]
**Describe the bug**
When running the command `prisma init --endpoint http://localhost:4466` as per the docs I get the following error:

```
Error: File or directory '/**/cli/node_modules/prisma-cli-core/dist/commands/init/boilerplate/datamodel.prisma' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:422:17)
    at findNativeAddonForStat (pkg/prelude/bootstrap.js:884:32)
    at statFromSnapshot (pkg/prelude/bootstrap.js:909:25)
    at Object.fs.lstatSync (pkg/prelude/bootstrap.js:950:12)
    at /snapshot/cli/node_modules/graceful-fs/polyfills.js:297:22
    at getStats (/snapshot/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:46:14)
    at startCopy (/snapshot/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:41:10)
    at Object.copySync (/snapshot/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:36:10)
    at Init.<anonymous> (/snapshot/cli/node_modules/prisma-cli-core/src/commands/init/index.ts:62:10)
    at step (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:42:23)
    at Object.f [as next] (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:23:53)
    at __awaiter (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:17:71)
    at new Promise (<anonymous>)
    at __awaiter (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:13:12)
    at Init.runInit (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:85:16)
    at Init.<anonymous> (/snapshot/cli/node_modules/prisma-cli-core/src/commands/init/index.ts:38:16)
    at step (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:42:23)
    at Object.f [as next] (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:23:53)
    at __awaiter (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:17:71)
    at new Promise (<anonymous>)
    at __awaiter (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:13:12)
    at Init.run (/snapshot/cli/node_modules/prisma-cli-core/dist/commands/init/index.js:60:16)
    at Function.<anonymous> (/snapshot/cli/node_modules/prisma-cli-engine/src/Command.ts:67:17)
    at step (/snapshot/cli/node_modules/prisma-cli-engine/dist/Command.js:32:23)
    at Object.f [as next] (/snapshot/cli/node_modules/prisma-cli-engine/dist/Command.js:13:53)
    at fulfilled (/snapshot/cli/node_modules/prisma-cli-engine/dist/Command.js:4:58)
Exiting with code: 1
```

**To Reproduce**
1. Run `prisma init --endpoint http://localhost:4466`
2. See error

**Expected behavior**
`prisma init` should run without errors

**Versions (please complete the following information):**
 - OS:`OS X High Sierra`
 - Prisma CLI: `prisma/1.17.0-beta.22 (darwin-x64) node-v10.4.1`

For some context - I had a previous version of the prisma CLI installed and I upgraded via homebrew. I've tried uninstalling and removing all traces of prisma and then reinstalling but it appears as if the prisma binary isn't compiling properly.

**Comment by marktani** [comment]
I think this is related to https://github.com/prisma/prisma/pull/3086

**Comment by covertbert** [comment]
> I think this is related to #3086

It seems likely indeed!

**Comment by divyenduz** [comment]
This is resolved and in stable (1.18) now. Please use `brew upgrade prisma` to upgrade.

**Comment by ripperdoc** [comment]
Installed from brew today and it seems to only be "1.17.0-beta.22" that is installed. Had to use npm to get the fixed 1.17.1 version.
